### PR TITLE
PWGGA/GammaConv: OmegaToPiZeroTask - added new background calculation

### PR DESCRIPTION
**Pull request by mhemmer-cern**
- Added new background calculation where two photons are rotated around a pi0 candidate to creat background pi0 candidates which are then combined with a third photon for an omega background candidate
- Added new train config to use this new method
- Added QA plot for this new method, where you can see the pi0 background candidates
- Changed fDoGammaSwappForBg in AliConversionMesonCuts from bool to an integer, which should not interfere with any of the existing use cases.

### CODE REVIEW

File: PWGGA/GammaConvBase/AliConversionMesonCuts.h [hunk 1]
```diff
@@ -311,7 +311,7 @@ class AliConversionMesonCuts : public AliAnalysisCuts {
     Bool_t      fDoJetPtMixing;                 ///< flag to enbale mixing by jet pt bins
     Bool_t      fDoSphericityMixing;            ///< flag to enable Sphericitymixing for meson bg estimation
     Bool_t      fUseTrackMultiplicityForBG;     ///< flag to use track multiplicity for meson bg estimation (else V0 mult)
-    Bool_t      fDoGammaSwappForBg;             ///< flag to use cluster swapping for background estimation
+    Int_t       fDoGammaSwappForBg;             ///< flag to use cluster swapping for background estimation
```
Comment (dmuhlhei) [inline PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: please increment the classdef

**Review by dmuhlhei — CHANGES_REQUESTED** [review]

File: PWGGA/GammaConvBase/AliConversionMesonCuts.h (hunk 1 above)
Comment (mhemmer-cern) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: Thanks for reminding me!
Updated both class defs now.

Comment (dmuhlhei) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: this request was only for conversionmesoncuts

Comment (dmuhlhei) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: for your user task it is not needed as you just add a transient member variable (//!)

Comment (mhemmer-cern) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: So when do you have to increment it?

Comment (dmuhlhei) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: Whenever you add/remove a non-transient member variable

Comment (dmuhlhei) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: ClassDef increment is now missing for MesonCuts - why did you touch AliConversionCuts? There are no real changes, but a ClassDef increment - please remove that from your PR

Comment (mhemmer-cern) [reply PWGGA/GammaConvBase/AliConversionMesonCuts.h:314]: Learned something new again. :)
Reverted the change for my task.

**Review by dmuhlhei — CHANGES_REQUESTED** [review]

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**